db/structure.sql

```diff
@@ -1,8 +1,8 @@
--- MySQL dump 10.13  Distrib 5.6.35, for osx10.11 (x86_64)
+-- MySQL dump 10.13  Distrib 5.6.41, for osx10.13 (x86_64)
 --
--- Host: localhost    Database: usasearch_development
```

Comment:
ah, I ran rake db migrate on development then test.

Reply:
@peggles2 , please re-run the migration so that it references `usasearch_development`. The reference to `usasearch_test` shouldn't cause any issues, but I'd rather not make unnecessary changes that will be reverted the next time a migration is run.

Reply:
Yeah that wont hurt anything its just stating that the migration was for test.

Reply:
on production it doesn't use development or test. Really doesn't matter.

Reply:
Yeah - I just don't it changing back and forth for no reason. Thanks for fixing that.